tests/src/test/scala/cats/tests/OptionTSuite.scala

```diff
@@ -285,10 +285,14 @@ class OptionTSuite extends CatsSuite {
     }
   }
 
-  test("OptionT.whenF[F, A] consistent with the same implementation of Option.when") {
-    val when = (c: Boolean, j: Int) => if (c) Some(j) else None
-    forAll { (li: List[Int], b: Boolean) =>
-      OptionT.whenF(b)(li).value should ===(li.map(when(b, _)))
+  test("OptionT.whenF[F, A] consistent with Option.when") {
+    def when[A]: (Boolean, A) => Option[A] = (c: Boolean, a: A) => if (c) Some(a) else None
```

Comment:
Not a blocker, but since the property refers to `Option.when`, it might be helpful to future readers to comment why it's inlined here (e.g., that it's not available before 2.13).